index.js

```diff
@@ -1,1496 +0,0 @@
-const { exit } = require('process');
-const readline = require('readline');
-const rl = readline.createInterface(process.stdin, process.stdout);
-
-function ask(questionText) {
-  return new Promise((resolve, reject) => {
-    rl.question(questionText, resolve);
-  });
-}
-
-function getRandomInt(max) {
-  return Math.floor(Math.random() * Math.floor(max));
-}
-
-start();
-
-async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n"); //compare this somehow lol
-  console.log('You entered: ' + secretNumber);
-  //let fInput = getRandomInt(100) // this is your first input variable
-  let firstResponse = await ask("Is your number " + 50 + " ?" + "\nY/N "); // first attempt at guess
-  let highOrLow = "Is it higher or lower?" //reusable 
-  
-
-  if (firstResponse === "N") { //first logic here??
-
-    console.log(highOrLow)
-  }
-  else {
-    console.log("Your number was " + 50 + "!" + "\nI guessed it in 1 try") //win
```

Comment:
If you were to use the `getRandomInt` function you could set it to a variable, this variable could be used in the `console.log()` to guess the number. You could also have it say "you guessed it in `variable` tries" and have this variable also increase programmatically